index.js

```diff
@@ -61,6 +61,7 @@ module.exports = {
         var dotFolders            = this.readConfig('dotFolders');
         var serverSideEncryption  = this.readConfig('serverSideEncryption');
         var batchSize             = this.readConfig('batchSize');
+        var defaultMimeType       = this.readConfig('defaultMimeType');
```

Comment:
Even though this is specified in the [README](https://github.com/ember-cli-deploy/ember-cli-deploy-s3#defaultmimetype) we don't have an actual default for it in the `defaultConfig` above. Worth adding it there?